### PR TITLE
Added python file to Installation.rst

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -72,7 +72,8 @@ You can also install PyAEDT from Conda-Forge with this command:
 
 PyAEDT remains compatible with IronPython and can be still used in the AEDT Framework.
 
-PyAnsys
+
+PyANSYS
```
Comment (PipKat) [inline doc/source/index.rst:76]: There should be no instance where we use PyANSYS. Ansys is always to be lower case except in copyright statements.

File: doc/source/index.rst [hunk 2]
```diff
@@ -93,7 +94,7 @@ Ansys Q3D Extractor, Ansys Siwave, and Ansys Icepak using electrical CAD (ECAD)
 Mechanical CAD (MCAD) workflows.
 
 In addition, AEDT includes direct links to the complete Ansys portfolio of thermal, fluid,
-and mechanical solvers for comprehensive multiphysics analysis.
+and Mechanical solvers for comprehensive multiphysics analysis.
```
Comment (PipKat) [inline doc/source/index.rst:97]: I was told that this was "mechanical" in the general sense and so should be lower case.

File: doc/source/index.rst [hunk 3]
```diff
@@ -1,13 +1,13 @@
 PyAEDT documentation  |version|
 ===============================
 
-|pyansys| |pypi| |PyPIact| |PythonVersion| |GH-CI| |codecov| |MIT| |black| |Anaconda| |pre-commit|
+|pyansys| |PyPI| |PyPIact| |PythonVersion| |GH-CI| |coverage| |MIT| |black| |Anaconda| |pre-commit|
```
Comment (PipKat) [inline doc/source/index.rst:4]: Note that the documentation for other PyAnsys libraries do not include this banner information. It's included only in the README. The then generally include what's in the "README" in the index by starting the include at a specified line.

File: doc/source/index.rst [hunk 4]
```diff
@@ -57,7 +57,7 @@ You can install PyAEDT on CPython 3.7 through 3.10 from PyPI with this command:
 
     pip install pyaedt
 
-To install PyAEDT with all extra packages (Matplotlib, NumPy, Pandas, and PyVista),
+To install PyAEDT with all extra packages (matplotlib, numpy, pandas, and pyvista),
```
Comment (PipKat) [inline doc/source/index.rst:60]: I'm OK with this change because it's referring to packages rather than organizations.

**Review by PipKat — APPROVED** [review]

File: doc/source/Getting_started/versioning.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    from pyaedt import settings
+
+    settings.use_grpc_api = False
+
+
+
+Compatibility between AEDT and interfaces
+=========================================
+
+The following table shows the supported versions of Ansys and the recommended interface for each one of them in PyAEDT.
```
Comment (MaxJPRey) [inline doc/source/Getting_started/versioning.rst:48]:
```suggestion
The following table shows the supported versions of Ansys EDT and the recommended interface for each one of them in PyAEDT.
```

File: doc/source/index.rst (hunk 1 above)
Comment (maxcapodi78) [reply doc/source/index.rst]: Thank you I've added to accept.txt

File: doc/source/index.rst [hunk 6]
```diff
@@ -166,7 +167,7 @@ Here are the main advantages that PyAEDT provides:
 - Compatibility with IronPython and CPython
 - Simplification of complex API syntax using data objects while
   maintaining PEP8 compliance.
-- Code reusability across different solvers
+- Code reusable across different solvers
```
[the remark this answers is not in the document]
Comment (maxcapodi78) [reply doc/source/index.rst]: Thank you I've added to accept.txt

**Pull request merged**